doc/api/n-api.md

```diff
@@ -286,41 +286,63 @@ listed as supporting a later version.
     <th scope="col">5</th>
     <th scope="col">6</th>
     <th scope="col">7</th>
+    <th scope="col">8</th>
   </tr>
   <tr>
     <th scope="row">v10.x</th>
     <td>v10.16.0</td>
     <td>v10.17.0</td>
     <td>v10.20.0</td>
     <td></td>
```

Comment:
```suggestion
    <td>v10.23.0</td>
```
node-api 7 was added to 10.23.0: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#2020-10-27-version-10230-dubnium-lts-richardlau